### PR TITLE
fix: bitvector.go function errors

### PR DESCRIPTION
**Pull request by hanksuper**
fix errors for some functions.

---
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/1906)

### CODE REVIEW

**Comment by acud** [comment]
Hello, thanks for the contribution!

Can you explain what is this exactly solving?

**Review by zelig — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1906)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @hanksuper)

File: pkg/bitvector/bitvector.go [hunk 1]
```diff
@@ -80,10 +80,10 @@ func (bv *BitVector) Unset(i int) {
 //
 // The argument must be the same as the bitvector length
 func (bv *BitVector) SetBytes(bs []byte) error {
-	if len(bs) != bv.len {
+	if len(bs)*8 != bv.len {
```
Comment (hanksuper) [inline pkg/bitvector/bitvector.go:83]: len(bs)   count in byte, but bv.len count in bit.

**Comment by hanksuper** [comment]
> Hello, thanks for the contribution!
> 
> Can you explain what is this exactly solving?



> Hello, thanks for the contribution!
> 
> Can you explain what is this exactly solving?

**Comment by hanksuper** [comment]
> Hello, thanks for the contribution!
> 
> Can you explain what is this exactly solving?

all fix about the same mistakes:  len(bs)   count in byte, but bv.len count in bit.

**Review by mrekucci — COMMENTED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1906)* status: all files reviewed, 1 unresolved discussion (waiting on @hanksuper)

___
*[pkg/bitvector/bitvector.go, line 114 at r1](https://reviewable.io/reviews/ethersphere/bee/1906#-Mb2U67M5EZ0650_pk0z:-Mb2U67NCYwakPE7qb4h:bihakag) ([raw file](https://github.com/ethersphere/bee/blob/8ca88975ba71c8cdb9c52e7355cb235d10be1603/pkg/bitvector/bitvector.go#L114)):*
> ```Go
> func (bv *BitVector) String() (s string) {
> 	for i := 0; i < bv.len; i++ {
> 		if bv.Get(i) {
> ```

We could benefit here from `strings.Builder`.

**Comment by acud** [comment]
@hanksuper the code which is being fixed is dead and not used anywhere. I would suggest to remove the bitvector methods which are not used. Can we convert this PR into a cleanup PR?

**Comment by acud** [comment]
closing due to inactivity

**Pull request closed without merging**